File: pinyougou-web-manager/src/main/webapp/js/service/orderService.js
// 定义服务层:
app.service("orderService",function($http){

	// this.search = function(page,rows,searchEntity){
	// 	return $http.post("../order/search.do?pageNum="+page+"&pageSize="+rows,searchEntity);
	// }
    this.searchOrder = function(){
        return $http.post("../order/searchOrder.do");
    }
    this.findOrderTotal = function(){
        return $http.post("../order/findOrderTotal.do");
    }

});

File: pinyougou-interface/src/main/java/cn/itcast/core/service/OrderService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.order.Order;
import entity.PageResult;
import vo.OrderVo;

import java.util.List;
import java.util.Map;

/**
 * Created by wang on 2019/4/29.
 */
public interface OrderService {
    void add(Order order);

    PageResult search(Integer pageNum, Integer pageSize, Order order);

    List<OrderVo> searchOrder();

    Integer findOrderTotal();
}
